src/lib/validations/sql.js

```diff
@@ -167,11 +167,30 @@ function checkTablePrefixes( tables ) {
 	}
 }
 
+const openFile = ( filename, flags = 'r', mode = 666 ) => new Promise( ( resolve, reject ) => {
+	fs.open( filename, flags, mode, ( err, fd ) => {
+		if ( err ) {
+			return reject( err );
+		}
+		resolve( fd );
+	} );
+} );
+
 export const validate = async ( filename: string, isImport: boolean = true ) => {
 	await trackEvent( 'import_validate_sql_command_execute', { isImport } );
 
+	let fd;
+
+	try {
+		fd = await openFile( filename );
+	} catch ( e ) {
+		console.log( chalk.red( 'Error: The file at the provided path is either missing or not readable.' ) );
```

Comment:
Thoughts on just making `Error:` red to improve readability?

```suggestion
		console.log( chalk.red( 'Error: ' ) + 'The file at the provided path is either missing or not readable.' );
```